src/ng/directive/ngModelOptions.js

```diff
@@ -0,0 +1,378 @@
+'use strict';
+
+/* global $DefaultModelOptionsProvider: true */
+
+/**
+ * @ngdoc provider
+ * @name $defaultModelOptionsProvider
+ * @description
+ *
+ * Here, you can change the default settings from which {@link ngModelOptions}
```

Comment:
It's fine by me. But then why do we need a public service for that? People will always be able to overwrite that.